src/backend/modules/user/user.service.js

```diff
@@ -1,17 +1,18 @@
 const { UserRepository } = require('./user.repository');
 const { DuplicateException } = require('../../common/httpException');
+const UserModel = require('../../model/user.model');
 
 class Service {
     constructor() {
-        this.userRepository = UserRepository;
+        this.repository = UserRepository;
     }
 
-    async createOne(body) {
-        const isUserExist = await this.userRepository.findByEmail(body.email);
+    async createOne(userDto) {
+        const isUserExist = await this.repository.findByEmail(userDto.email);
         if (isUserExist) {
-            throw new DuplicateException(`User (${body.email}) is already existed`);
+            throw new DuplicateException(`User (${userDto.email}) is already existed`);
         } else {
```

Comment:
You don't have to use the *else* statement here. Because if user exists, the *throw* method will also return, so the rest statement won't execute.